src/domain/resource/resourceReservationButton/ResourceReservationButton.js

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Button from 'react-bootstrap/lib/Button';
+import get from 'lodash/get';
+import moment from 'moment';
```

Comment:
Isn't it so that you should use moment.tz ?

Reply:
Looking at the code in Varaamo I see that I'm probably confused. Good to verify that no bugs arise if you set your computer to a time in another timezone